go.mod

```diff
@@ -50,3 +50,5 @@ require (
 	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22 // indirect
 	upper.io/db.v3 v3.7.1+incompatible
 )
+
+replace github.com/pingcap/tidb-tools => github.com/pingcap/tidb-tools v4.0.0-beta.0.20200921090810-52b9534ff3d5+incompatible
```

Comment:
if we don't use `go mod replace` here, the `tidb-tools` always get the following hash
`github.com/pingcap/tidb-tools v4.0.6-0.20200828085514-03575b185007+incompatible`